### DISCUSSION TITLE
Gestione Energia

### DISCUSSION THREAD

**Discussion by vinny74man** [indomus/forum#1529] [HUB: Home Assistant]
Ciao a tutti sto facendo leggere da HA vari dispositivi in modbus che leggono la potenza istantanea e i consumi "contatore" di una fabbrica, per la visualizzazione tutto ok HA si comporta benissimo con i registri modbus, lo scopo sarebbe quello di rilevare i consumi di una data apparecchiatura per intervalli di tempo, cerco di spiegarmi meglio, ho l'apparecchiatura 1 voglio sapere quanto ha consumato di energia da lunedì a mercoledì, giusto per fare un esempio, l'intervallo potrebbe essere qualsiasi, c'è una scheda che mi faccia selezionare data di inizio e data di fine e mi visualizzi il consumo, cioè faccia la sottrazione tra il consumo letto della data maggiore e quello della data inferiore, al momento vedo i grafici e diciamo che da li potrei fare io il calcolo, ma mi servirebbe qualcosa di più immediato.

Grazie

**Comment by Scialla** [comment]
Prova a guardare questo:
https://indomus.it/componenti/componente-home-assistant-utility-meter/

**Reply by vinny74man** [reply]
Lo avevo visto questo sensore, però sembra più statico di quello che serve a me, in quanto devi definire prima il ciclo di contabilizzazione e poi lui fa il calcolo, ma a me non serve in questo modo, quello che servirebbe a me dovrebbe permettermi di impostare una data di inizio e fine e fare il calcolo in base hai dati archiviati dal sensore energia.  Pensi sia possibile creare qualcosa ad hoc per questo tipo di esigenza, ed eventualmente come fare.

Grazie
